src/components/Main/MainView/MessageInput/use/wordCompleter.ts

```diff
@@ -0,0 +1,60 @@
+import store from '@/store'
+import createTree from '@/lib/trieTree'
+import { ComputedRef, WritableComputedRef } from 'vue'
+
+function getCurrentWord(elm: HTMLTextAreaElement, text: string) {
```

Comment:
```suggestion
const getCurrentWord = (elm: HTMLTextAreaElement, text: string) => {
```
ほかのところこっちで書いてあるからこっちで統一したい 🙇